server/cp/ingest/parser/cp_onclusive.py

```diff
@@ -47,11 +47,12 @@ def parse(self, content, provider=None):
                                 )
                     if subject["scheme"] == "onclusive_event_types":
                         for event_item in event_types_cvs:
-                            if event_item["event_name"] == subject["name"]:
+                            if event_item["name"] == subject["name"]:
```

Comment:
I think there will be another cv `event_types` with types which will be used by cp and the names won't match necessarily. so in this `cp_event_types` we need a mapping from cp events to this other cv (which we don't have yet, you can create a new one with a few types).
so in `cp_event_types` we need to have: `{"name": "onclusive event name", "cp_type": "qcode from event_types cv"}`